### DISCUSSION TITLE
this.incrementSceneReference is not a function while using push,replace

### DISCUSSION THREAD

**Discussion by abhisonar** [ViroCommunity/viro#211] [Q&A]
I am implementing indoor navigtion using ViroReact . I am trying to push scene after tracking image but getting 
**this.incrementSceneReference is not a function**

`const onPlaceAnchorFound = place => {
    console.log('anchor found', place);
    console.log('props', props);
    getPOIsFromBuilding();
    // navigation.navigate('IndoorLocation');
    props.arSceneNavigator.push('IndoorLocation', IndoorLocation);
  };`
  
  `<ViroARScene>
      {placeData.map(place => {
        return (
          <ViroARImageMarker
            target={place.marker_slug}
            onAnchorFound={() => onPlaceAnchorFound(place)}>
            <ViroBox position={[0, 0, 0]} scale={[0.1, 0.1, 0.1]} />
          </ViroARImageMarker>
        );
      })}
    </ViroARScene>`

**Comment by BrunoCabelo** [comment]
Did you find any solution to this problem?
